app/assets/javascripts/site_wide/ad.js

```diff
@@ -2,7 +2,6 @@
 //   var iframe = document.getElementById('topad');
 //   iframe.src = iframe.src;
 // },150000);
-// window.setInterval(function() {
-//   var iframe = document.getElementById('footad');
-//   iframe.src = iframe.src;
-// },60000);
\ No newline at end of file
+window.setInterval(function() {
```

Comment:
Identifier 'foot_ad' is not in camel case.<br>Identifier 'top_ad' is not in camel case.<br>'googletag' is not defined.<br>'foot_ad' is not defined.<br>'top_ad' is not defined.